### ISSUE TITLE
Error: invalid input 'encoder_attention_mask'

### ISSUE THREAD

**Issue by spencekim** [xenova/transformers.js#23]
I'm trying to run a slightly larger model (https://huggingface.co/facebook/bart-large-cnn). It's a Bart model, and I converted it to .onnx successfully using your script. Size comes out to ~1gb.

I get this error when running the summarization and text2text generation pipelines:

```
Error: invalid input 'encoder_attention_mask'
    at eval (ort-web.min.js?d8cf:6:446899)
    at Array.forEach (<anonymous>)
    at e.OnnxruntimeWebAssemblySessionHandler.run (ort-web.min.js?d8cf:6:446819)
    at InferenceSession.run (inference-session-impl.js?f23d:91:1)
    at sessionRun (models.js?a626:34:1)
    at seq2seq_forward (models.js?a626:111:1)
    at async Function.forward (models.js?a626:971:1)
    at async seq2seqRunBeam (models.js?a626:168:1)
    at async Function.runBeam (models.js?a626:964:1)
    at async Function.generate (models.js?a626:562:1)
```

**Comment by xenova** [comment]
Could you provide the command used to export?

**Comment by spencekim** [comment]
Yep that's the exact command I used. If you'd like, I could upload the files to your HuggingFace repo (https://huggingface.co/Xenova/transformers.js/tree/main) to make sure they look ok.

**Comment by xenova** [comment]
No worries - I'll convert it on my side. Then it's most likely something I'd need to fix. The strange thing is, other model types (with the same model type, i.e., "bart") need the encoder attention mask (vs. what you are showing, which doesn't).

**Comment by xenova** [comment]
Okay yep I was able to reproduce it on my side. As I said above, it's very strange that some bart models need it, while others dont... I'll make it work either way :)

**Comment by xenova** [comment]
Got it working :) 
```[{"summary_text":"The Eiffel Tower is the tallest structure in Paris. It is 324 metres (1,063 ft) tall, about the same height as an 81-storey building."}]```

Will push changes soon.

**Comment by xenova** [comment]
Looks like the commit automatically closed the issue. Whoops.

Can you confirm the changes work, and if so, I will push this in version 1.2.6

**Comment by spencekim** [comment]
Sorry for the late response. It works great!

Side note, any plans to set up Typescript types, and/or support onnxruntime-node? Both would be super useful. I'd be open to helping implement it as well.

**Comment by xenova** [comment]
> Sorry for the late response. It works great!

Great!

> Side note, any plans to set up Typescript types, and/or support onnxruntime-node? Both would be super useful. I'd be open to helping implement it as well.

Absolutely! We currently have a couple of people working on the typescript side of things (https://github.com/xenova/transformers.js/pull/26 and https://github.com/xenova/transformers.js/pull/28)

For onnxruntime-node, I suppose it would be a question of how to support both web and node versions of onnxruntime without duplicating code. If you have any ideas, or know how to do that, feel free to open up a PR!
